### ISSUE TITLE
why my loss comes to negative?

### ISSUE THREAD

**Issue by koestlerWang** [BrikerMan/Kashgari#110]
when i use kashgari to do NER task, the loss change from positive to negative.
But loss is generated from softmax,which is positive always.

**Comment by BrikerMan** [comment]
> when i use kashgari to do NER task, the loss change from positive to negative.
> But loss is generated from softmax,which is positive always.

Which model you are using？

**Comment by koestlerWang** [comment]
> > when i use kashgari to do NER task, the loss change from positive to negative.
> > But loss is generated from softmax,which is positive always.
> 
> Which model you are using？
chinese_L-12_H-768_A-12 and BLSTMCRFModel

**Comment by BrikerMan** [comment]
Here are some bugs in the crf layer, which cause negative loss. Will fix later.

**Comment by koestlerWang** [comment]
问一下大神，这个bug fix了吗

**Comment by BrikerMan** [comment]
还没有，这个是我用的 CRF 实现的 bug，具体信息看这里 https://github.com/keras-team/keras-contrib/issues/253，但不影响使用。
